### ISSUE TITLE
Object.dirname is deprecated.

### ISSUE THREAD

**Issue by michael-gracy** [mschuchard/linter-terraform-syntax#12]
Argument to `path.dirname` must be a string
```
Object.dirname (/Users/michael.gracy/Applications/Atom.app/Contents/Resources/app.asar/src/electron-shims.js:9:10)
Object.lint (/Users/michael.gracy/.atom/packages/linter-terraform-syntax/lib/main.js:38:35)
a.factory (/Users/michael.gracy/.atom/packages/atom-ide-ui/modules/atom-ide-ui/pkg/atom-ide-diagnostics/lib/services/LinterAdapter.js:309:42)
a._callFactory (/Users/michael.gracy/.atom/packages/atom-ide-ui/node_modules/rxjs/bundles/Rx.min.js:107:23)
a.tryDefer (/Users/michael.gracy/.atom/packages/atom-ide-ui/node_modules/rxjs/bundles/Rx.min.js:106:442)
new a (/Users/michael.gracy/.atom/packages/atom-ide-ui/node_modules/rxjs/bundles/Rx.min.js:106:383)
```

**Comment by mschuchard** [comment]
Is this on Windows?

**Comment by michael-gracy** [comment]
I'm sorry that I haven't replied. This is on MacOS Sierra

**Comment by mschuchard** [comment]
It is ok. The root cause was discovered in the linked issue above in my other Atom package for `ansible-lint`. Check that out now for how to avoid the issue, and a fix is forthcoming.

**Comment by mschuchard** [comment]
Fixed in 741362f.
